### DISCUSSION TITLE
Can we have team names present in the "Schedule" and "Bracket Editor" menus in the osu tournament client?

### DISCUSSION THREAD

**Discussion by 0x84f** [ppy/osu#17540] [Ideas] [answered]
Piggybacking this idea off of discussion #17539 
Currently if you do not set an acronym for a team the bracket editor fields and the schedule look completely empty, which made me think it was some sort of a bug with the client and not an issue of lacking acronyms.
Is there possibility to have the full team names used for the brackets and schedules instead of their acronyms, similar to the "Team Intro" screen? Also if there isn't a flag specified can the names just be across the entire "tile" instead having a blank space [like this](https://i.imgur.com/BzUeJTb.png).

**Comment by peppy** [comment, marked as answer]
Can you please show screenshots of what you're seeing currently? You can drag and drop images into the text window to include them, rather than linking externally.

**Reply by 0x84f** [reply]
Sorry, include images of what, I'm not sure I understood. I am talking about the names on the bracket and schedule menus.
If you don't add acronyms and if you don't want to use country flags the bracket looks [like this](https://i.imgur.com/tTBBuqq.png), and the schedule [like this](https://i.imgur.com/NneBhgC.png).
I was wondering if full team names could be used instead of acronyms and if flags can be removed from those screens if they are not needed.

**Reply by peppy** [reply]
For future posts, could you please drag images into the text box rather than use imgur links. It makes it very hard to actually see what you mean.

**Reply by peppy** [reply]
So in your case, your team names are so long they wouldn't fit in the first place. You're going to see something like "hi alg..." and "Alham...". Is that what you're looking for?

**Reply by 0x84f** [reply]
I was thinking if it was possible for those boxes to be dynamically scaled with the names instead of always using the acronyms?

**Reply by peppy** [reply]
That's probably not going to happen.

**Reply by 0x84f** [reply]
In that case I guess we can close this off? I can't mark your reply as an answer unfortunately.
